docs/clients/uavcan_node.rst

```diff
@@ -0,0 +1,132 @@
+| Sub ID | Short Name             | Access         | Data Type | Unit             | Note                                                                                                                                                                                                                                                                                                                    |
++========+========================+================+===========+==================+=========================================================================================================================================================================================================================================================================================================================+
+| 0      | uavcan_node_id         | get, set, save | uint32    | Number [1, 127]  | This ID is used by the module to uniquely identify itself on the DroneCAN bus. The module must be power cycled after changing the ID. Each module on the bus must have a unique ID. While the IDs can be a number between 1 and 127, it is recommended to avoid using 1 as this is commonly used by flight controllers. |
++--------+------------------------+----------------+-----------+------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 1      | uavcan_esc_index       | get, set, save | uint32    | Number [0, 19]   | This is used to identify the module when ESC commands are set on the DroneCAN bus. Each ESC/module on the bus requires a unique index to identify which commands are intended for it. The ESC indexes range from 0 to 19, with the first module starting at index 0.                                                    |
++--------+------------------------+----------------+-----------+------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 2      | zero_behavior          | get, set, save | uint32    | Number [0, 2]    | This determines how the module reacts to receiving a zero setpoint from a DroneCAN Raw Command when DroneCAN is bypassing arming. 0 \= coast, 1 \= Brake, 2 \= Normal Controller                                                                                                                                        |
```

Comment:
Have we used enum as a unit before? That might be the right sort of thing to use for zero_behavior